crates/swc/tests/errors/lints/check-private-property/2/private_props.js

```diff
@@ -0,0 +1,19 @@
+class Foo {
```

Comment:
I think this should be in 

https://github.com/swc-project/swc/tree/7d40208377133b0fcd153c5bfcf47f6b489113c1/crates/swc/tests/exec